src/json-schema/partial/common.json

```
@@ -157,7 +157,8 @@
                     "MaxAttempts": {
                         "type": "integer",
                         "description": "A non-negative integer representing the maximum number of retry attempts. (Default: 3)",
```

Do we want to make reference to the maximum value in the property description?

---

It doesn't hurt, though the validation message already tells you this.
```suggestion
                        "description": "A non-negative integer representing the maximum number of retry attempts. (Default: 3, Maximum: 99999999)",
```